doc/FEATURES_RELEASE_2.md

```diff
@@ -41,7 +41,6 @@ DDI. Press `Submit`. The system should display the DDI information
 for the DDI found, including whether its disposition is `Good` or `Bad`,
 the timing window and units of this timing window, as well as the drugs involved.
```

Comment:
I think we should add the exact expected information here - like we do with the other tests.